BitPatty/gctGenerator
# V3

Replaces the static page with [VuePress](https://vuepress.vuejs.org/)

TODO's:
- Before Merge: Set Github Pages branch to gh-pages :warning:
- Test Codes  ✔
- Test Stage Loader❓
- Compatibility
    - Chrome ✔
    - Firefox✔
    - Edge (Chromium) ✔
    - IE 11 ✔
    - ~Edge ❓~ (Not relevant anymore)
- Metadata ✔
- Pipeline updates
    - build ✔
    - deploy ❓ (SHOULD work)
- Check whether all links and sections are re-linked properly ✔
